org.envirocar.app/res/color/dashboard_indicator_selector.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-
-    Copyright (C) 2013 - 2019 the enviroCar community
-
-    This file is part of the enviroCar app.
-
-    The enviroCar app is free software: you can redistribute it and/or
-    modify it under the terms of the GNU General Public License as published
-    by the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    The enviroCar app is distributed in the hope that it will be useful, but
-    WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
-    Public License for more details.
-
-    You should have received a copy of the GNU General Public License along
-    with the enviroCar app. If not, see http://www.gnu.org/licenses/.
-
--->
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="@color/cario_color_primary_dark" android:state_enabled="false" />
-    <item android:color="@color/cario_warning_red" android:state_enabled="true" />
```

Comment:
Maybe, another solution would be to bind the background color to another state, e.g. `state_selected`. So it would be unnecessary to change the background tint within the `DashboardFragment` class.